src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -2843,7 +2843,7 @@
     <value>A case-insensitive lookup for resource file "{0}" in assembly "{1}" found multiple entries. Remove the duplicates or specify the exact case.</value>
   </data>
   <data name="MissingManifestResource_NoNeutralAsm" xml:space="preserve">
-    <value>Could not find any resources appropriate for the specified culture or the neutral culture.  Make sure "{0}" was correctly embedded or linked into assembly "{1}" at compile time, or that all the satellite assemblies required are loadable and fully signed.</value>
+    <value>Could not find any resources appropriate for the specified culture or the neutral culture. {0} not found among the resources {2}. Make sure "{0}" was correctly embedded or linked into assembly "{1}" at compile time, or that all the satellite assemblies required are loadable and fully signed.</value>
```

Comment:
Since we have a chance to reword the message, how about something like

> Could not find the resource "{0}" among the resources {2} embedded in the assembly "{1}", nor among the resources in any satellite assemblies for the specified culture. Perhaps the resources were embedded with an incorrect name.

Perhaps with quoting instead of square brackets, eg

> Could not find the resource "MyApp.Form1.resources" among the resources "Form1.resources", "MyImage.bmp", "Sound.wav" embedded in assembly "MyApp", nor among the resources in any satellite assemblies for the specified culture.  Perhaps the resources were embedded with an incorrect name.

Reply:
Ok, I'll do that.